docs/sync/2026-04-20_inbound_event_queue_design.md

```
@@ -0,0 +1,876 @@
+sessionManager.timelineEvents.listen((event) {
+  if (event.roomId != currentRoomId) return;
+  unawaited(queue.enqueueLive(event));
+});
+```
+
+This replaces today's `MatrixStreamSignalBinder.start` body and its
+five overlapping signal paths. Zero flag negotiation.
+
+### 7.2 Limited-sync bridge producer
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)